Sources/MapboxNavigation/JunctionView.swift

```diff
@@ -56,7 +56,7 @@ public class JunctionView: UIImageView {
         
         guard let guidanceView = quaternaryInstruction?.components.first else { return }
         
-        if case .guidanceView(let guidanceViewImageRepresentation, _) = guidanceView {
+        if case .guidanceView(let guidanceViewImageRepresentation, _, _) = guidanceView {
```

Comment:
Does this code remain applicable to all guidance views, or only the conventional kind that has previously been supported? (Would the kind be `nil` in that case?)

Reply:
This code remains applicable for all guidance views, in case if kind is not present it'll be `nil` by default.